### PR TITLE
Removed redundant comparison

### PR DESCRIPTION
**Pull request by mkstowe**
Removed redundant comparison in if statement as described in #8571.

### CODE REVIEW

**Comment by duncanspumpkin** [comment]
We are currently working on bug fixes for 0.2.6 this will be looked at afterwards.

File: src/openrct2/localisation/StringIds.h [hunk 1]
```diff
@@ -1927,8 +1927,9 @@ enum
     STR_SHORTCUT_SHOW_RECENT_MESSAGES = 2522,
     STR_SHORTCUT_SHOW_MAP = 2523,
     STR_SHORTCUT_SCREENSHOT = 2524,
+	STR_X_PERSON_DIED_ON_X = 2525,
```
Comment (tupaschoal) [inline src/openrct2/localisation/StringIds.h:1930]:
```suggestion
    STR_X_PERSON_DIED_ON_X = 2525,
```

File: data/language/en-GB.txt [hunk 2]
```diff
@@ -1858,6 +1858,7 @@ STR_2521    :Show staff list
 STR_2522    :Show recent messages
 STR_2523    :Show map
 STR_2524    :Screenshot
+STR_2225    :{RED}{COMMA16} person has died in an accident on {STRINGID}
```
Comment (tupaschoal) [inline data/language/en-GB.txt:1861]: Add it to the end, don't reuse previous ids.

File: src/openrct2/ride/Vehicle.cpp [hunk 3]
```diff
@@ -5204,7 +5204,14 @@ static void vehicle_kill_all_passengers(Vehicle* vehicle)
...
         ride->FormatNameTo(gCommonFormatArgs + 2);
-        news_item_add_to_queue(NEWS_ITEM_RIDE, STR_X_PEOPLE_DIED_ON_X, vehicle->ride);
+        if (numFatalities == 1)
+        {
+			news_item_add_to_queue(NEWS_ITEM_RIDE, STR_X_PERSON_DIED_ON_X, vehicle->ride);
+        }
+        else
+        {
+			news_item_add_to_queue(NEWS_ITEM_RIDE, STR_X_PEOPLE_DIED_ON_X, vehicle->ride);
+        }
```
Comment (tupaschoal) [inline src/openrct2/ride/Vehicle.cpp:5214]:
```suggestion
        news_item_add_to_queue(NEWS_ITEM_RIDE, numFatalities == 1 ? STR_X_PERSON_DIED_ON_X : STR_X_PEOPLE_DIED_ON_X, vehicle->ride);
```
I prefer it like this, but you can also assign to a variable based on this and have just one call for `add_to_queue`

**Comment by mkstowe** [comment]
Reverted back to the original commit.

**Comment by tupaschoal** [comment]
CI is failing, so I'm making this WIP. Can you tend to it, please?

**Comment by duncanspumpkin** [comment]
Please delete the dl variable and this is fine to merge.

**Comment by duncanspumpkin** [comment]
I checked the assembly and it looked agreeable.

**Pull request merged**